### PR TITLE
setup skeleton zebra-utils crate for independent binaries

### PR DESCRIPTION
**Pull request by yaahc**
This is intended to be a starting point for https://github.com/ZcashFoundation/zebra/issues/564

### CODE REVIEW

**Comment by yaahc** [comment]
I went ahead and setup the example for passing args through, the current interface looks like this:

<pre><font color="#06989A"><b>src/bin/zebra-checkpoints</b></font> on <font color="#75507B"><b> zebra-utils</b></font> <font color="#CC0000"><b>[$] </b></font>via <font color="#CC0000"><b>🦀 v1.44.1</b></font> took <font color="#C4A000"><b>16s</b></font> 
<font color="#CC0000"><b>❯</b></font> cargo run -- --help
<font color="#4E9A06"><b>    Finished</b></font> dev [unoptimized + debuginfo] target(s) in 0.07s
<font color="#4E9A06"><b>     Running</b></font> `/home/jlusby/git/zfnd/zebra/target/debug/zebra-checkpoints --help`
zebra-utils 0.1.0

USAGE:
    zebra-checkpoints [FLAGS] [-- &lt;zcli-args&gt;...]

FLAGS:
    -h, --help       Prints help information
    -t, --testnet    Use the test network
    -V, --version    Prints version information

ARGS:
    &lt;zcli-args&gt;...    Passthrough args for `zcash-cli`</pre>

File: Cargo.toml [hunk 1]
```diff
@@ -9,6 +9,7 @@ members = [
         "zebra-rpc",
         "zebra-client",
         "zebra-test",
+	"zebra-utils",
```
Comment (hdevalence) [inline Cargo.toml]: match indentation?

File: zebra-utils/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+[package]
+name = "zebra-utils"
+version = "0.1.0"
+authors = ["Jane Lusby <jane@zfnd.org>"]
```
Comment (hdevalence) [inline zebra-utils/Cargo.toml]: this should probably have the same generic author as the rest of the crates

**Review by hdevalence — CHANGES_REQUESTED** [review]
LGTM modulo comments!

**Review by hdevalence — APPROVED** [review]

**Pull request merged**